build/cmake/CMakeLists.txt

```diff
@@ -7,9 +7,12 @@
 # in the COPYING file in the root directory of this source tree).
 # ################################################################
 
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 3.0.0...3.13.2 FATAL_ERROR)
```

Comment:
Why increasing the version requirement from v2.8.9 ?
That change can be problematic, because not all systems are up to date.